test/sql/blob/test_blob.cpp

```diff
@@ -0,0 +1,268 @@
+#include "catch.hpp"
+#include "test_helpers.hpp"
+#include "duckdb/main/prepared_statement.hpp"
+
+#include <fstream>
+
+using namespace duckdb;
+using namespace std;
+
+TEST_CASE("BLOB null and empty values", "[blob]") {
```

Comment:
Please add some tests with using blobs in various operations (GROUP BY blob, JOIN tables on blobs, ORDER BY blob).